exercises/two-bucket/canonical-data.json

```diff
@@ -96,6 +96,50 @@
         "goalBucket": "two",
         "otherBucket": 2
       }
+    },
+    {
+      "uuid": "449be72d-b10a-4f4b-a959-ca741e333b72",
+      "description": "Not possible to reach the goal",
+      "property": "measure",
+      "input": {
+        "bucketOne": 6,
+        "bucketTwo": 15,
+        "goal": 5,
+        "startBucket": "one"
+      },
+      "expected": {
+        "error": "impossible"
+      }
+    },
+    {
+      "uuid": "aac38b7a-77f4-4d62-9b91-8846d533b054",
+      "description": "With the same buckets but a different goal, then it is possible",
+      "property": "measure",
+      "input": {
+        "bucketOne": 6,
+        "bucketTwo": 15,
+        "goal": 9,
+        "startBucket": "one"
+      },
+      "expected": {
+        "moves": 10,
```

Comment:
Been so long that I wrote this case that I probably should make sure it's correct. I know I could just get the code to verify it but we should also do it by hand in case there's a bug in the code

Bucket amounts:

1. 6 0
1. 0 6
1. 6 6
1. 0 12
1. 6 12
1. 3 15
1. 3 0
1. 0 3
1. 6 3
1. 0 9
```

So yes I believe 10 is correct